core/src/main/codegen/templates/Parser.jj

```diff
@@ -5678,8 +5679,19 @@ SqlNode NamedFunctionCall() :
             args.remove(0);
         }
     )
+    [
+        (
+            <IGNORE> { s1 = span(); } <NULLS> {
+                ignoreNulls = SqlLiteral.createBoolean(true, s1.end(this));
+            }
+        |
+            <RESPECT> { s1 = span(); } <NULLS> {
+                ignoreNulls = SqlLiteral.createBoolean(false, s1.end(this));
+            }
+        )
+    ]
     {
-        call = createCall(qualifiedName, s.end(this), funcType, quantifier, args);
+        call = createCall(qualifiedName, s.end(this), funcType, quantifier, ignoreNulls, args);
```

Comment:
Sorry I haven't tested the code but Is it possible to make `{ RESPECT | IGNORE } NULLS` clause an individual operator? There are already some operators for similar agg-specific clauses such as `OVER` / `FILTER` / `WITHIN GROUP`.

Reply:
Thank you for your suggestion. With my understanding, if doing like that, the first operand of `RexOver` may become `RESPECT_NULLS(AGG)`  or `IGNORE_NULLS(AGG)` instead of `AGG`. It looks like more reasonable and I would like to try it.

Reply:
@zhztheplayer I spent some time on figuring out whether your proposal works. It seems that it can't work due to unable to find the right function which ignores nulls. Because the function has been resolved to the function which respects nulls when creating `RESPECT_NULLS(AGG)` unless we use different name for the functions which seems a little hacky.